lib/service/skills/MockOccupationsData.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:illinois/model/occupation/Occupation.dart';
```

Comment:
I think since you have set up your occupations service to send requests to O*NET, we don't need this class anymore.